assets/js/plugin-check-admin.js

```diff
@@ -265,12 +266,28 @@
 
 		// Render remaining files with only warnings.
 		for ( const file in warnings ) {
+			errorWarningMessage = true;
 			renderFileResults( file, [], warnings[ file ] );
 		}
 
 		resultsContainer.innerHTML += renderTemplate(
 			'plugin-check-results-complete'
 		);
+
+		const resultsMessage = document.getElementById(
+			'plugin-check__message'
+		);
+		if ( resultsMessage ) {
+			const resultsMessageChild = resultsMessage.querySelector( 'p' );
+			if ( errorWarningMessage ) {
+				resultsMessage.classList.remove( 'notice-success' );
+				resultsMessage.classList.add( 'notice-error' );
+				resultsMessageChild.innerHTML += ' ' + pluginCheck.errorMessage;
+			} else {
+				resultsMessageChild.innerHTML +=
```

Comment:
```suggestion
				resultsMessage.classList.add( 'notice-success' );
				resultsMessage.classList.remove( 'notice-error' );
				resultsMessageChild.innerHTML +=
```

If the message goes from a error to success if run multiple times. Let's be overly careful here.

Reply:
Thanks @spacedmonkey for the feedback. Based on the other feedback i use other approach to solve the issue. Please take a look. Thanks!